features/step_definitions/oauth_cluster.rb

```diff
@@ -0,0 +1,30 @@
+# setup tier_down to restore scc after scenario end;
+Given /^oauth cluster is restored after scenario$/ do |cluster|
```

Comment:
btw this step is not used in the scenario. It is preferable to add new steps when they are used instead together with unrelated scenarios.

Reply:
Sorry, my mistake to add this step.